mpf/commands/__init__.py

```diff
@@ -98,9 +111,13 @@ def parse_args(self):
 
         if len(self.argv) > 1 and self.argv[1].startswith('-'):
             self.argv.insert(1, None)
-
         args, remaining_args = parser.parse_known_args(self.argv[1:])
-        machine_path = self.get_machine_path(args.machine_path)
+
+        # If the production arg was pulled here, re-insert it to the remaining args
+        # so that the game parser also knows to run in production mode.
+        if args.production:
```

Comment:
A quirk of the `parse_known_args()` method is that it consumes the known args and only passes unknown args to the next consumer. In this case I need both _init.py_ and _game.py_ to consume the `-P` arg.

I looked through the parser documentation and didn't see any way to keep an arg in both places, so here I re-add it so that _game.py_ can consume it. Totally open to a better approach!